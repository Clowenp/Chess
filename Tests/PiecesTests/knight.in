n 0 0 2 2
n 4 4 5 6
n 7 4 8 6
n 4 0 1 1
