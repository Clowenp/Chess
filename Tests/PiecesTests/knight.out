i am a knight at [0,0]
my possible paths are: 
< [1,2] >
< [2,1] >
< >
< >
< >
< >
< >
< >
the move to [2,2] is NOT possible
i am a knight at [4,4]
my possible paths are: 
< [5,6] >
< [6,5] >
< [6,3] >
< [5,2] >
< [3,2] >
< [2,3] >
< [2,5] >
< [3,6] >
the move to [5,6] is possible
i am a knight at [7,4]
my possible paths are: 
< >
< >
< >
< >
< [6,2] >
< [5,3] >
< [5,5] >
< [6,6] >
the move to [8,6] is NOT possible
i am a knight at [4,0]
my possible paths are: 
< [5,2] >
< [6,1] >
< >
< >
< >
< >
< [2,1] >
< [3,2] >
the move to [1,1] is NOT possible
